packages/@aws-cdk/aws-codebuild/package.json

```diff
@@ -1,100 +1,102 @@
-    "@aws-cdk/aws-codecommit": "^0.25.0",
-    "@aws-cdk/aws-codepipeline-api": "^0.25.0",
-    "@aws-cdk/aws-ecr": "^0.25.0",
-    "@aws-cdk/aws-events": "^0.25.0",
-    "@aws-cdk/aws-iam": "^0.25.0",
-    "@aws-cdk/aws-kms": "^0.25.0",
-    "@aws-cdk/aws-s3": "^0.25.0",
-    "@aws-cdk/cdk": "^0.25.0"
-  },
-  "engines": {
-    "node": ">= 8.10.0"
-  }
-}
\ No newline at end of file
+	"name": "@aws-cdk/aws-codebuild",
+	"version": "0.25.0",
+	"description": "CDK Constructs for AWS CodeBuild",
+	"main": "lib/index.js",
+	"types": "lib/index.d.ts",
+	"jsii": {
+		"outdir": "dist",
+		"targets": {
+			"java": {
+				"package": "software.amazon.awscdk.services.codebuild",
+				"maven": {
+					"groupId": "software.amazon.awscdk",
+					"artifactId": "codebuild"
+				}
+			},
+			"dotnet": {
+				"namespace": "Amazon.CDK.AWS.CodeBuild",
+				"packageId": "Amazon.CDK.AWS.CodeBuild",
+				"signAssembly": true,
+				"assemblyOriginatorKeyFile": "../../key.snk"
+			},
+			"sphinx": {}
+		}
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/awslabs/aws-cdk.git"
+	},
+	"scripts": {
+		"build": "cdk-build",
+		"watch": "cdk-watch",
+		"lint": "cdk-lint",
+		"test": "cdk-test",
+		"integ": "cdk-integ",
+		"pkglint": "pkglint -f",
+		"package": "cdk-package",
+		"awslint": "cdk-awslint",
+		"cfn2ts": "cfn2ts"
+	},
+	"cdk-build": {
+		"cloudformation": "AWS::CodeBuild"
+	},
+	"nyc": {
+		"statements": 74,
+		"lines": 74
+	},
+	"keywords": [
+		"aws",
+		"cdk",
+		"constructs",
+		"codebuild"
+	],
+	"author": {
+		"name": "Amazon Web Services",
+		"url": "https://aws.amazon.com",
+		"organization": true
+	},
+	"license": "Apache-2.0",
+	"devDependencies": {
+		"@aws-cdk/assert": "^0.25.0",
+		"@aws-cdk/aws-sns": "^0.25.0",
+		"@aws-cdk/aws-sqs": "^0.25.0",
+		"aws-sdk": "^2.259.1",
+		"cdk-build-tools": "^0.25.0",
+		"cdk-integ-tools": "^0.25.0",
+		"cfn2ts": "^0.25.0",
+		"pkglint": "^0.25.0"
+	},
+	"dependencies": {
+		"@aws-cdk/assets": "^0.25.0",
+		"@aws-cdk/assets-docker": "^0.25.0",
+		"@aws-cdk/aws-cloudwatch": "^0.25.0",
+		"@aws-cdk/aws-codecommit": "^0.25.0",
+		"@aws-cdk/aws-codepipeline-api": "^0.25.0",
+		"@aws-cdk/aws-ecr": "^0.25.0",
+		"@aws-cdk/aws-ec2": "^0.25.0",
+		"@aws-cdk/aws-events": "^0.25.0",
+		"@aws-cdk/aws-iam": "^0.25.0",
+		"@aws-cdk/aws-kms": "^0.25.0",
+		"@aws-cdk/aws-s3": "^0.25.0",
+		"@aws-cdk/cdk": "^0.25.0"
+	},
+	"homepage": "https://github.com/awslabs/aws-cdk",
+	"peerDependencies": {
+		"@aws-cdk/assets": "^0.25.0",
+		"@aws-cdk/assets-docker": "^0.25.0",
+		"@aws-cdk/aws-cloudwatch": "^0.25.0",
+		"@aws-cdk/aws-codecommit": "^0.25.0",
+		"@aws-cdk/aws-codepipeline-api": "^0.25.0",
+		"@aws-cdk/aws-ecr": "^0.25.0",
+		"@aws-cdk/aws-ec2": "^0.25.0",
+		"@aws-cdk/aws-events": "^0.25.0",
+		"@aws-cdk/aws-iam": "^0.25.0",
+		"@aws-cdk/aws-kms": "^0.25.0",
+		"@aws-cdk/aws-s3": "^0.25.0",
+		"@aws-cdk/cdk": "^0.25.0"
+	},
+	"engines": {
+		"node": ">= 8.10.0"
+	}
+}
```

Comment:
Uhm, what happened here? Can you add the EC2 dependency, while leaving the formatting as it was? Thanks!

Reply:
Sorry - Wasn't ready for review - I've reverted the formatting (inconsistent tab size)